provisioning/Samples/device/SymmetricKeyWithOperationalCertificateSample/SymmetricKeyWithOperationalCertificateSample.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
I've been removing this from projects I create. I'm not a huge fan yet. You?

Reply:
This has been forcing me to declare nullable types as nullable, which in turn has made me add more null checks to my applications. It's been interesting.